app/views/pages/partials/_calendar.html.erb

```diff
@@ -1,37 +1,37 @@
-<div id="calendar" data-controller="calendar" data-calendar-events-value="<%= @events %>">
-  <%= vite_image_tag "images/illustrations/210.png", class: "background" %>
+  <div id="calendar" data-controller="calendar" data-calendar-events-value="<%= @events %>">
```

Comment:
I see that here the only difference is an indentation of the code to the right.
Could you please just indent it back one space to the left? (lines 1 to 37)